front/app/modules/commercial/native_surveys/components/NativeSurveyForm/NativeSurveyForm.tsx

```diff
@@ -0,0 +1,168 @@
+import useLocale from 'hooks/useLocale';
+import { isNilOrError } from 'utils/helperUtils';
+
+const NativeSurveyForm = () => {
+  const locale = useLocale();
+
+  const renderers = [
+    { tester: inputControlTester, renderer: InputControl },
+    { tester: textAreaControlTester, renderer: TextAreaControl },
+    { tester: checkboxControlTester, renderer: CheckboxControl },
+    { tester: singleSelectControlTester, renderer: SingleSelectControl },
+    { tester: multiSelectControlTester, renderer: MultiSelectControl },
+    { tester: WYSIWYGControlTester, renderer: WYSIWYGControl },
+    { tester: descriptionControlTester, renderer: DescriptionControl },
+    { tester: topicsControlTester, renderer: TopicsControl },
+    { tester: titleControlTester, renderer: TitleControl },
+    { tester: imageControlTester, renderer: ImageControl },
+    { tester: attachmentsControlTester, renderer: AttachmentsControl },
+    { tester: clCategoryTester, renderer: CLCategoryLayout },
+    { tester: orderedLayoutTester, renderer: OrderedLayout },
+    { tester: locationControlTester, renderer: LocationControl },
+    { tester: dateControlTester, renderer: DateControl },
+    { tester: userPickerControlTester, renderer: UserPickerControl },
+    { tester: multilocInputTester, renderer: MultilocInputLayout },
+    { tester: orderedLayoutTester, renderer: OrderedLayout },
+  ];
+
+  const dummyData = {
+    name: 'John Doe',
+  };
+
+  // Dummy data mocking an API response. Once back office is set up, replace this with an API call.
+  const dummyAPIResponse = {
+    json_schema_multiloc: {
+      en: {
+        type: 'object',
+        properties: {
+          nameEnglish: {
+            type: 'string',
+            minLength: 1,
+          },
+        },
+        required: ['name'],
+      },
+      'fr-BE': {
+        type: 'object',
+        properties: {
+          nameFrench: {
+            type: 'string',
+            minLength: 1,
+          },
+        },
+        required: ['name'],
+      },
+    },
+    ui_schema_multiloc: {
+      en: {
+        type: 'VerticalLayout',
+        elements: [
+          {
+            type: 'Control',
+            scope: '#/properties/nameEnglish',
+          },
+        ],
+      },
+      'fr-BE': {
+        type: 'VerticalLayout',
+        elements: [
+          {
+            type: 'Control',
+            scope: '#/properties/nameFrench',
+          },
+        ],
+      },
+    },
+  };
+
+  return (
+    <>
+      <Box
+        display="flex"
+        flexWrap="wrap"
+        justifyContent="center"
+        width="100%"
+        id="e2e-native-survey-form"
+      >
+        <Title marginTop="48px" width="100%" textAlign="center">
+          Survey Title
+        </Title>
+        <Box
+          padding="40px 40px 30px"
+          borderRadius="4px"
+          width="652px"
+          background="white"
+        >
+          {!isNilOrError(locale) && (
```

Comment:
Potentially an improvement for later. We could return null when there is no locale or show a loader 🤔